pkg/controller/dynamic/dynamic_controller_integration_test.go

```diff
@@ -368,7 +372,7 @@ func testUpdate(ctx context.Context, t *testing.T, testContext testrunner.TestCo
 	// Update resource from test data
 	updateUnstruct := testContext.UpdateUnstruct.DeepCopy()
 	if updateUnstruct == nil {
```

Comment:
As we've added the check above, is this check still needed?

Reply:
Good point; removed it!